crates/bench/src/spacetime_raw.rs

```diff
@@ -24,17 +23,16 @@ impl BenchDatabase for SpacetimeRaw {
     }
     type TableId = TableId;
 
-    fn build(in_memory: bool, fsync: bool) -> ResultBench<Self>
+    fn build(in_memory: bool, _fsync: bool) -> ResultBench<Self>
```

Comment:
Should we deal with the unused _fsync here?

Reply:
It does have some significance for impls of `BenchDatabase` for other databases.